src/Microsoft.Identity.Client/Platforms/net45/WebUIFactory.cs

```diff
@@ -35,15 +34,15 @@ internal class WebUIFactory : IWebUIFactory
     {
         private PlatformParameters parameters;
 
-        public IWebUI CreateAuthenticationDialog(IPlatformParameters inputParameters)
+        public IWebUI CreateAuthenticationDialog(IPlatformParameters inputParameters, UIBehavior behavior)
         {
             parameters = inputParameters as PlatformParameters;
             if (parameters == null)
             {
                 throw new ArgumentException("parameters should be of type PlatformParameters", nameof(parameters));
             }
 
-            if (parameters.UseHiddenBrowser)
+            if (behavior.Equals(UIBehavior.Never))
```

Comment:
We have dev utility apps and E2E apps for .net as well which can get scenario coverage for platform specific stuff. Core functionality can be tested with high confidence using unit tests. So.. for now it is a low priority.